movement-test/Makefile

```diff
@@ -1,24 +1,22 @@
 # pod = Pineapple of Doom
 PROG = pod
 
-GBDK_N = ../gbdk-n
-CC = $(GBDK_N)/bin/gbdk-n-compile.sh
-LD = $(GBDK_N)/bin/gbdk-n-link.sh
-MR = $(GBDK_N)/bin/gbdk-n-make-rom.sh
+export SDCCDIR=/usr  # might not work for everyone
```

Comment:
Can this be overridden if need be?

Reply:
Would it be cleaner to just expect users to set the `SDCCDIR` env var then?

Reply:
But yes, you can override this with another path (this var is only required to compile gbdk-2020)